embeddings/task/lightning_task/lightning_task.py

```diff
@@ -86,14 +94,61 @@ def fit_predict(
         self,
         data: HuggingFaceDataModule,
         predict_subset: LightingDataModuleSubset = LightingDataModuleSubset.TEST,
+        run_name: Optional[str] = None,
     ) -> Dict[str, nptyping.NDArray[Any]]:
         if not self.model:
             raise self.MODEL_UNDEFINED_EXCEPTION
-        self.fit(data)
+        self.fit(data, run_name=run_name)
         dataloader = data.get_subset(subset=predict_subset)
         assert isinstance(dataloader, DataLoader)
-        return self.predict(dataloader=dataloader)
+        result = self.predict(dataloader=dataloader)
+        return result
 
     @abc.abstractmethod
     def build_task_model(self) -> None:
         pass
+
+    def get_lightning_loggers(
+        self,
+        run_name: Optional[str] = None,
+        use_wandb: bool = True,
+        use_tensorboard: bool = True,
+        use_csv: bool = True,
+        wandb_project: Optional[str] = None,
+        wandb_entity: Optional[str] = None,
+    ) -> List[LightningLoggerBase]:
+        """Based on configuration, provides pytorch-lightning loggers' callbacks."""
+        loggers: List[LightningLoggerBase] = []
+
+        if use_tensorboard:
+            loggers.append(
+                pl_loggers.TensorBoardLogger(
+                    name=run_name,
+                    save_dir=str(self.output_path.joinpath("tensorboard")),
+                )
+            )
+
+        if not use_wandb and (wandb_project or wandb_entity):
+            raise ValueError(
+                "`wandb_project` or `wandb_entity` was configured but use_wand is set to false."
+            )
+
+        if use_wandb:
+            save_dir = self.output_path.joinpath("wandb")
```

Comment:
do we handle the case when `use_wandb` is True but `wandb_project` and `wandb_entity` were not configured?